File: src/dart/ext4_optimizer/lib/reader.dart
String reader(int position, int language) {
  List<String> dictionaryEng = [
		"Choose a Option",
		"Optimize a ext4 partition",
		"Exit to the shell",
		"Please select a partition" ,
		"Press Enter to continue...",
		"Optimize",
		"Exit"
  ];

	List<String> dictionaryEsp = [
    "Seleccione una opcion",
		"Optimizar una particion de tipo ext4",
		"Salir al shell",
		"Por favor selecciona una partition",
		"Presione Enter para continuar...",
		"Optimizar",
		"Salir"
  ];

	if(language == 1) {
		return dictionaryEng[position];
	} else {
		return dictionaryEsp[position];
	}
}

File: src/rsyncer/go/lib/reader.go
package lib

func Reader(position int, languaje int) string {

	DICTIONARY_ENG := [3]string{
		"Please write your source directory",
		"Please write your destination directory to copy",
		"Press Enter to continue...",
	}

	DICTIONARY_ESP := [3]string{
		"Por favor escriba su directorio de origen",
		"Por favor escriba su directorio de destino",
		"Presione Enter para continuar...",
	}

	if languaje == 1 {
		return DICTIONARY_ENG[position]
	} else {
		return DICTIONARY_ESP[position]
	}
}
